### ISSUE TITLE
固定随机数似乎不能复现结果

### ISSUE THREAD

**Issue by bbbxixixixi** [PaddlePaddle/PaddleX#668]
能定的随机数都定了
import random
import paddle.fluid as fluid
seed = 2021
fluid.default_startup_program().random_seed = seed
fluid.default_main_program().random_seed = seed
random.seed(seed)
np.random.seed(seed)
paddle.framework.seed(seed)

但是重复训练，各种参数，训练数据都没变，结果不同
![111](https://user-images.githubusercontent.com/24725986/113909120-e8098e00-9809-11eb-8e9c-9dc3956980c5.png)
![22](https://user-images.githubusercontent.com/24725986/113909131-eb047e80-9809-11eb-93a7-aa1957fbbd6a.png)

**Comment by bbbxixixixi** [comment]
用的是
model = pdx.seg.HRNet(
    num_classes=N_CLASSES,
    width=WIDTH,
    class_weight=weights, 
    input_channel=4
)

model.train(
        num_epochs=EPOCHS,
        train_dataset=train_dataset,
        train_batch_size=BATCH_SIZE, 
        eval_dataset=eval_dataset,
        learning_rate=LR,
        log_interval_steps=20,
        save_interval_epochs=SAVE_INTERVAL,
        save_dir=SAVE_DIR,
        pretrain_weights='/home/aistudio/HRNet_W64_C_pretrained',
        # resume_checkpoint='/home/aistudio/Models/HRNet64/epoch_200', #需要从断点继续训练时可以将该参数设置为模型保存的路径
        use_vdl=True)

**Comment by jiangjiajun** [comment]
目前没有提供固定复现的方法，我们会在2.0版本中提供此功能（当前除了数据增强中存在随机性，参数初始化也存在随机性，因此固定随种数并不能复现结果）
